src/braket/default_simulator/cirq/program_context.py

```diff
@@ -0,0 +1,88 @@
+from typing import Tuple, List
+
+import cirq
+import numpy as np
+from braket.ir.jaqcd.program_v1 import Results
+from cirq import Circuit
+
+from braket.default_simulator.openqasm.program_context import AbstractProgramContext
+
+CIRQ_GATES = {
+    "i": cirq.I,
+    "h": cirq.H,
+    "x": cirq.X,
+    "y": cirq.Y,
+    "z": cirq.Z,
+    "cnot": cirq.CNOT,
+    "cz": cirq.CZ,
+    "s": cirq.S,
+    "t": cirq.T,
+    "cphaseshift": cirq.cphase,
+    "rx": cirq.rx,
+    "ry": cirq.ry,
+    "rz": cirq.rz,
+    "swap": cirq.SWAP,
+    "iswap": cirq.ISWAP,
+    # "xx": cirq.XX,
+    # "yy": cirq.YY,
+    # "zz": cirq.ZZ,
+    "ccnot": cirq.CCNOT,
+    "cswap": cirq.CSWAP,
+    "ccz": cirq.CCZ,
+    "ccx": cirq.CCX,
+    "measure": cirq.MeasurementGate,
+}
+
+
+class ProgramContext(AbstractProgramContext):
```

Comment:
Big fan of the structure here, very cool that you can do this without touching the interpreter